mmv1/templates/terraform/examples/access_context_manager_service_perimeter_granular_controls.tf.erb

```diff
@@ -0,0 +1,55 @@
+resource "google_access_context_manager_access_policy" "access-policy" {
```

Comment:
Is this to add a test or an example in the web documentation? Either way, it needs to be registered in the corresponding resource.yaml file. See step 5 in https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-a-create-test. If it's documentation only, you'll need to add `skip_test: true` like https://github.com/GoogleCloudPlatform/magic-modules/blob/216fffd274b08af2e3f718d3d8aeb8cb6410b420/mmv1/products/cloudids/Endpoint.yaml#L37